flexmeasures/data/models/reporting/profit.py

```diff
@@ -158,7 +167,7 @@ def _compute_report(
 
             results.append(
                 {
-                    "name": "cashflow",
+                    "name": "profit",
```

Comment:
Shouldn't this name reflect the sign that was set on initialization? Would that have any effect with regards to how intuitive the config and/or parameters would look?

Reply:
Sure! Actually, the name here is used as for annotation. The name is used in the `PandasReporter` to link an ouput_required to an output.